README.md

```diff
@@ -56,6 +56,38 @@ jobs:
       - run: java -cp java HelloWorldApp
 ```
 
+Publising to an Apache Maven Repository:
+```yaml
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@master
+      with:
+        java-version: 1.8
+        server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
+        username: ${{ github.actor }} # username for server authentication
+        password: ${{ github.token }} # password or token for authentication
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
+    - name: Publish to GitHub Packages Apache Maven
+      run: mvn deploy
+    - name: Set up Apache Maven Central
+      uses: actions/setup-java@master
+      with: # running setup-java again overwrites the settings.xml
+        java-version: 1.8
+        server-id: maven
+        username: maven_username
+        password: ${{ secrets.MAVEN_CENTRAL_TOKEN }} # password from secrets store
+    - name: Publish to Apache Maven Central
```

Comment:
Could you also add an example using Gradle

Reply:
I was going to do Gradle in a follow up PR. Lemme try to wrap this up and if I can I'll add an example as well.

Reply:
Another PR with a Gradle example is fine. As long as we eventually have one.

Reply:
Ok, added a Gradle example in 203550b
I have a demo build of gradle here https://github.com/pied-piper-inc/fib-gradle/commit/2b7855275aaa4153ba7f9fe64b3e9ef3b8272817/checks?check_suite_id=346412071